### PR TITLE
feat(storybook): deprecate CSS/Native components

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**  
In move towards the final removal of Native(css) components for the Tegel package we start with separating them in Storybook. 

**Solving issue**  
Fixes:[DTS-1469](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1469)

**How to test**  
Pull down the branch and build it and run storybook and click around and make sure that everything works. 

[DTS-1469]: https://tegel.atlassian.net/browse/DTS-1469?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tegel/src/components/stepper/sdds-stepper.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { formatHtmlPreview } from '../../utils/utils';
```
Comment (timrombergjakobsson) [inline tegel/src/components/stepper/sdds-stepper.stories.tsx:1]: Changed to ```.tsx``` instead.

**Review by mJarsater — APPROVED** [review]
🦄

**Review by theJohnnyMe — APPROVED** [review]
LGTM ;) Rock it!

**Review by mJarsater — CHANGES_REQUESTED** [review]
The deprecation needs to be communicated in some way before this is merged.

**Comment by mJarsater** [comment]
This needs a section in the migration docs.

**Review by mJarsater — APPROVED** [review]
🚀

**Review by mJarsater — CHANGES_REQUESTED** [review]
Looks like this needs a rebase

**Comment by timrombergjakobsson** [comment]
> Looks like this needs a rebase

yep! Will do asap!

**Comment by mJarsater** [comment]
> > Looks like this needs a rebase
> 
> yep! Will do asap!

No rush! Im thinking we'll hold off on the announcement until tomorrow! ✌️

**Review by mJarsater — COMMENTED** [review]
Nice! Do you wanna do the same thing for the Header and Side menu in this PR also?

**Comment by timrombergjakobsson** [comment]
> Nice! Do you wanna do the same thing for the Header and Side menu in this PR also?

Sure thing!

**Review by mJarsater — APPROVED** [review]
LGTM!

**Pull request merged**